src/main/java/com/google/devtools/build/lib/analysis/AnalysisUtils.java

```diff
@@ -203,17 +203,16 @@ public static TopLevelTargetsAndConfigsResult getTargetsWithConfigs(
     // configurations
     // for deps including transitions. So to satisfy its API we resolve transitions and repackage
     // each target as a Dependency (with a NONE transition if necessary).
```

Comment:
nit: should we update the `repackage each target as a Dependency` comment? Maybe we can just eliminate this whole sentence now, since the explicit typing makes the requirements clearer?

Reply:
Done.